starter-code/assets/css/modules.css

```diff
@@ -0,0 +1,413 @@
+
+h2.alt {
+    color: #888;
+}
+
+h2.alt strong {
+    color: #666;
+}
+header > p {
+    margin: 1em 0 0 0;
+}
+strong, b {
+    font-weight: 300;
+    color: #666;
+}
+
+em, i {
+    font-style: italic;
+}
+a:hover {
+    color: #E27689;
+    border-bottom-color: rgba(255, 255, 255, 0);
+}
+
+p, ul, ol, dl, table {
+    margin-bottom: 2em;
+}
+
+br.clear {
+    clear: both;
+}
+.container {
+    margin: 0 auto;
+    max-width: 100%;
+    width: 1400px;
+}
+
+.row {
```

Comment:
I recommend putting comments before each section of styling. That way you know what element this row belongs to.